napari/layers/points/points.py

```diff
@@ -1034,11 +1034,8 @@ def selected_data(self, selected_data):
             k: np.unique(v[index], axis=0) for k, v in self.properties.items()
         }
         n_unique_properties = np.array([len(v) for v in properties.values()])
-        self.current_properties = {}
-        # only include the properties that common across all selected points
-        for n, prop in zip(n_unique_properties, properties):
-            if n == 1:
-                self.current_properties[prop] = properties[prop]
+        if np.all(n_unique_properties == 1):
+            self.current_properties = properties
```

Comment:
Wait, can't we fix this in the same way as last time, but perhaps with more checks?

Reply:
The challenge is that if two points with different properties are selected, it isn't clear how to resolve the conflicting properties to set `current_properties`without causing an error when subsequently adding new points. For example, if we have the points layer below

```python
data = [[100, 200], [200, 200]]
properties = {
    'point_class': np.array(['A', 'B']),
    'radius': np.array([6, 6]),
}

layer = Points(
    data,
    face_color = 'point_class'
    properties=properties)
)

# select both points
layer.selected_data = [0, 1]
```
As we had it before, we give `current_properties`the properties that all of the selected points have in common. That is, 

```python
>>> layer.current_properties
{'radius': np.array([6])}
```
If we then add a point, we will get the error [described by Nick above](https://github.com/napari/napari/pull/933#issuecomment-580944251), because `layer.current_properties` does not have a `point_class` value (the selected points had different values for `point_class`).

I had considered resolving the conflicting properties by taking the first of the conflicting values (or something like that), but that didn't seem intuitive to the user (i.e., they wouldn't easily be able to figure out how the property is chosen).

I think another option could be to have the user define a fall back value for the properties, but I think we should avoid adding another property if we can.

It seemed like the most straightforward thing was to revert to the original behavior where it just doesn't overwrite the value of `Points.current_properties` if the selected points conflict (it's how the other properties work). All of that being said, I am open to other suggestions! What do you think, @jni?

Reply:
I think not overwriting is fine for now. I think we can revisit this when the think about how to represent the properties in the GUI / change properties for new points

Reply:
That works for me. @jni, do you have any ideas or a strong preference against merging as is?

Reply:
Yeah, ok, I see the issue clearly now. We can revisit later.